CHANGES.rst

```diff
@@ -33,6 +33,13 @@ extract_1d
 - Updated to propagate SRCTYPE keyword during extraction of MIRI LRS
   fixed-slit inputs that are in `SlitModel` form. [#6212]
 
+- Fix bug when creating cubes using outlsput_type=channel [#6138]
```

Comment:
this line is a (munged) duplicate of line 19 above and should be removed (it applies to cube_build, not extract_1d). Probably got messed up during conflict resolution.